Application/Commands/CreateWorkEntryCommandTests.cs

```diff
@@ -15,13 +15,16 @@ public async Task CreateWorkEntryAsync_ShouldSaveNewWorkEntryToDb()
 
         var mockClaimsProvider = GetMockClaimsProvider();
 
-        var createWorkEntryCommand = new CreateWorkEntryCommand(context, mockClaimsProvider);
+        var mockAssignmentsApi = GetMockAssignmentsApi();
+
+        var createWorkEntryCommand = new CreateWorkEntryCommand(context, mockClaimsProvider, mockAssignmentsApi);
```

Comment:
The creation logic becomes more and more complicated here. Consider using DI for CreateWorkEntryCommand instantiation instead of creating it manually